README.md

```diff
@@ -1 +1,152 @@
-# capstone
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>The Capstone project</b></h3>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+# 📖 [The Capstone Project] <a name="Capstone Project"></a>
+
+**[Capstone]** is a project to build a website to show a future Environmental Protection Global submit
+
+## 🛠 Built With <a name="built-with">Visual Studio</a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">Visual Studio</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://expressjs.com/">Github pages</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>No database was  used</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">NA</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **[Mobile version with Mobile menu]**
+- **[A Navbar with a banna]**
+- **[Speakers section dynamically generated by JS]**
+- **[Desktop Version]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://enocol.github.io/capstone/about.html)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+- create a local directory on your computer.
+- Open a Terminal and run the following code
+
+git clone https://github.come/enocol/capstone.git
+
+Note: You must have git running on your computer.
+
+### Prerequisites
+
+In order to run this project you need:
+
+### Setup
+
+Clone this repository to your desired folder:
+
+### Install
+
+Install this project with:
+
+### Usage
+
+To run the project, execute the following command:
+open the project in your refered code Editors and run a local live server.
+
+### Run tests
+
+To run tests, run the following command:
+
+### Deployment
+
+You can deploy this project using:
```

Comment:
- [ ] Please add the relevant information to each section highlighted above. Because right now it is difficult for someone to understand how your project works, what are the pre-requisite and what to install first in order to run your project or do collaboration in your project. Kindly try to make your `README` file more professional.